### DISCUSSION TITLE
VS Code: Emmet inserts nodes with the `className` attribute instead of `class`

### DISCUSSION THREAD

**Discussion by angry-meow** [solidjs/solid#2238] [Q&A]
Hi community!

If I type `section.my-class` and press `Tab`, Emmet in VS Code inserts `<section className="my-class"></section>` which is okay for React but kinda incorrect for Solid. The expected result would be `<section class="my-class"></section>`.

The file has the `*.tsx` extension.

How to fix that behavior?

**Comment by Nebula-Developer** [comment]
bump

**Comment by rwxae** [comment]
Still an issue. Moreover, the problem appears in other editors too. I encounter it in Zed.

**Comment by c727** [comment]
use this in `.vscode/settings.json`
```json
{
  "emmet.includeLanguages": {
    "javascriptreact": "html",
    "typescriptreact": "html"
  },
}
```
